### PR TITLE
Update tr.lang

### PR DESCRIPTION
**Pull request by LKHN**

### CODE REVIEW

File: META-INF/com/google/android/aroma/langs/tr.lang [hunk 1]
```diff
@@ -27,10 +27,10 @@
 
 
 ### ArchiDroid strings
-archidroid.apps.adaway.desc=Belirli aralıklarla sunucuyu güncelleyen, ArchiDroid'in Android için Açık kaynaklı olan reklam engelleyicisidir
+archidroid.apps.adaway.desc=Open source ad blocker for Android. Comes preconfigured to update hosts in ArchiDroid's Adblock periodically
```
Comment (JustArchi) [inline META-INF/com/google/android/aroma/langs/tr.lang:30]: Why back to english?

File: META-INF/com/google/android/aroma/langs/tr.lang [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
 archidroid.apps.cooltool.desc=Cihazın üstünde faydalı bilgileri (örn CPU kullanımı) gösteren isteğe göre değiştirilebilir uygulama
 archidroid.apps.esfileexplorer.desc=Android için çok iyi bir yedek dosya yöneticisi
-archidroid.apps.fdroid.desc=Özel Google Play Store eklentileriyle Android platformu için ücretsiz ve açık kaynak kodlu uygulamaların Kurulabilir kataloğu
+archidroid.apps.fdroid.desc=Installable catalogue of free and open source applications for the Android platform. Great addition to proprietary google store
```
Comment (JustArchi) [inline META-INF/com/google/android/aroma/langs/tr.lang:33]: Same here

File: META-INF/com/google/android/aroma/langs/tr.lang (hunk 1 above)
Comment (lkhn) [inline META-INF/com/google/android/aroma/langs/tr.lang:30]: I don`t know may be i was pressed wrong button :( Turkish translate is Perfect dont back English please

**Comment by JustArchi** [comment]
I can't divide this commit into two, I only accept what you commited ;). Please make additional commit which reverts these two strings back to turkish if you wish.

**Comment by lkhn** [comment]
ok i understand and allready corrected two stings, may be i press wrong button becouse i am new github :)

**Pull request merged**